app/models/comment.rb

```diff
@@ -1,2 +1,13 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] The same here, Great job creating a method that updates the comments counter for a post. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, 
- 🤔  Your code might look like this 👇🏻 :

```ruby
# app/models/comment.rb

     after_save : update_comments_counter
   
     private

      def update_comments_counter
        post.increment!(:comments_counter)
      end
```